.gitignore

```diff
@@ -6,4 +6,7 @@ _build/
 dist/
 MANIFEST
 Axelrod.egg-info
-.hypothesis/eval_source/
+.hypothesis/
```

Comment:
> To be honest, I haven't looked into hypothesis at all and simply added this to .gitignore because I kept accidentally committing test files.

Let's leave it as you have it here and I'll look in to making the db persist on travis. :) :+1: